newrelic/resource_newrelic_synthetics_monitor_script.go

```diff
@@ -42,10 +45,11 @@ func resourceNewRelicSyntheticsMonitorScript() *schema.Resource {
 							Required:    true,
 							Description: "The monitor script location name",
 						},
-						"hmac": {
+						"vse_password": {
```

Comment:
Hmm, I could be missing some context here, but are we fully replacing the `hmac` attribute? If so, this will be a breaking change. 

To make this backwards compatible, we could add the `vse_password` attribute while still keeping `hmac`, and then we can add the `Deprecated` schema option on the `hmac` attribute to let folks know it's being removed/replaced.

Reply:
@sanderblue Yeah this would be a breaking change. Will having the two values conflict? Only one should be set.

Reply:
I would add both, make it clear in the description/docs, and do validation in the structure like here: https://github.com/newrelic/terraform-provider-newrelic/blob/main/newrelic/structures_newrelic_nrql_alert_condition.go#L94